code/game/objects/items/weaponry.dm

```diff
@@ -613,7 +613,6 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 		var/mob/living/carbon/human/L = M
 		if(L && L.dna && L.dna.species)
 			L.dna.species.stop_wagging_tail(M)
-	if(user.a_intent != INTENT_HARM && ((user.zone_selected == BODY_ZONE_PRECISE_MOUTH) || (user.zone_selected == BODY_ZONE_PRECISE_EYES) || (user.zone_selected == BODY_ZONE_HEAD)))

```

Comment:
seems like this removes the ability to slap nonhumans

Reply:
It does, yeah

Reply:
goddamn it I'm useless I wrote slapping code in the first place and I still fuck it up I'll never make a pr without at least two problems
but yeah I fixed it